Makefile

```diff
@@ -13,7 +13,10 @@ DC_FILE=.docker/docker-compose.yml
 GOOS ?= $(shell go env GOOS)
 REPO ?= "$(shell go list -m)"
 VERSION ?= "$(shell git describe --tags 2>/dev/null | sed 's/^v//')$(shell if [ "$(GOOS)" = "windows" ]; then echo "_unsupported"; fi)"
-BUILD_FLAGS = "-X '$(REPO)/pkg/config.Version=$(VERSION)'"
+COMMITDATE ?= $(shell git show --no-patch --no-notes --pretty=%ci | xargs -I{} date -u -d '{}' +%Y%m%d%H%M%S)
+COMMITHASH = $(shell git show --no-patch --no-notes --pretty=%H | sed -r 's/(.{12}).*/\1/')
```

Comment:
Builds on Windows?

Reply:
Haven't tested it, but don't need to add unsupported suffix here.